python/scripts/rfi_toggle_cron.py

```diff
@@ -0,0 +1,57 @@
+import datetime
+import requests
+import json
+import time
+
+#Wait until the correct UTC time (deals with daylight savings time)
+while(datetime.datetime.utcnow().time() < datetime.time(19,10)):
+    time.sleep(5)
+
+downtime = 60 #minutes
+
+#Endpoint parameters
+url = 'http://csBfs:54323/toggle-rfi-zeroing'
+headers = {'content-type': 'application/json', 'Accept-Charset': 'UTF-8'}
+
+#Payload
+payload = dict()
+payload['rfi_zeroing'] = False
+
+#Turn Off
+on = True
+try:
+    r = requests.post(url, data=json.dumps(payload), headers=headers)
+    if(not r.ok):
+        print("RFI CRONJOB: Something went wrong in the request.")
+    else:
+        on = False
+        print("RFI CRONJOB: Request sent")
+except:
+    print("RFI CRONJOB: Failure to Contact Kotekan Master, is it running?")
+
+#If we successfully turned RFI Zeroing off
+if(not on):
```

Comment:
This logic seems wrong if the connection above failed, i.e. if we don't connect and hit the except clause, we then continue with `on = True` which means we never attempt to turn excision back on, even when we are at the correct time to do so.